modules/serverless-supported-channel-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-supported-channel-types_{context}"]
+= Supported channel types
+
+Currently, {ServerlessProductName} only supports `InMemoryChannel` kind channels for development use, as part of the Knative Eventing Technology Preview.
```

Comment:
Would `type` be better here instead of `kind`? That's how it was used in the next sentence, and I think it makes more sense.